lib/membrane/pipeline/action.ex

```diff
@@ -89,6 +89,19 @@ defmodule Membrane.Pipeline.Action do
   Changes the playback state of the pipeline to the chosen one.
   """
 
+  @type reply_t :: {:reply, message :: any}
+  @typedoc """
+  Action that replies to a `Pipeline.call/2`. Can be returned only from the `handle_call/3` callback, in
+  which context the caller reference is available, with the `:from` key.
```

Comment:
```suggestion
  Action that replies to a `Pipeline.call/2`. Can be returned only from the `handle_call/3` callback, in
  which context the caller reference is available, under the `:from` key.
```

Also, add the full reference to the `Pipeline.call` and `handle_call` so they were clickable in docs

Reply:
done